wafer/schedule/templates/wafer.schedule/full_schedule.html

```diff
@@ -41,7 +41,7 @@
             <td colspan="{{ item.colspan }}" rowspan="{{ item.rowspan }}" class="unavailable"></td>
          {% else %}
             {# Add item details #}
-            <td colspan="{{ item.colspan }}" rowspan="{{ item.rowspan }}" class="{{ item.item.css_class|default:'schedule' }}">
+            <td colspan="{{ item.colspan }}" rowspan="{{ item.rowspan }}" class="{{ item.item.css_class|default:'schedule' }}" class="{{ item.item.get_talk_css_class|default:'not_talk' }}">
```

Comment:
I wanted to avoid clashing with anything created as a talk type. but arguably creating a talk type named "None" falls under the "consenting adults" provisions.